internal/cmd/operator-sdk/bundle/validate/validate.go

```diff
@@ -81,11 +81,6 @@ func (c *bundleValidateCmd) addToFlagSet(fs *pflag.FlagSet) {
 
 	fs.StringVarP(&c.outputFormat, "output", "o", internal.Text,
```

Comment:
Can you add some help text saying something like
```
Note: output format types containing "alphaX" are subject to change and not covered by guarantees of stable APIs.
```

Reply:
Done